docs/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * Added configurable width for debug lines. ([#1016])
 * Added `TextureMetadata::srgb_scale()` for default texture metadata with nearest filter. ([#1023])
 * Added motivation to use Amethyst over gluing the building blocks yourself in the book. ([#1057])
+* Added `Config::load_bytes` for reading configuration from raw bytes.
```

Comment:
You're missing a link to this PR

Reply:
Added

Reply:
You also need to add the actual link further down :wink:

Reply:
Oh damn sorry, I forget that it is markdown, not magic. Done